pkg/metrics/client.go

```diff
@@ -0,0 +1,40 @@
+package metrics
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/dcos/dcos-cli/pkg/httpclient"
+)
+
+// Client is a metrics client for DC/OS.
+type Client struct {
+	http *httpclient.Client
+}
+
+// NewClient creates a new metrics client.
+func NewClient(baseClient *httpclient.Client) *Client {
+	return &Client{
+		http: baseClient,
+	}
+}
+
+// Node returns the units of a certain node.
+func (c *Client) Node(mesosID string) (Node, error) {
```

Comment:
```suggestion
func (c *Client) Node(mesosID string) (*Node, error) {
```

Would return a pointer here. Makes it more consistent throughout other API calls and instead of `Node{}` you can return `nil`.